### DISCUSSION TITLE
Equipd Bible v4.1.3

### DISCUSSION THREAD

**Discussion by de1mat** [Equipd/Bible#5] [Announcements]
It has been a while since the last update, thanks for your patience - I hope you are still enjoying the app.

This update fixes a few issues and adds some new features.

# New In-App Browser

- Much easier to install new Bible EPUB files and Schedules.
- Support for browser bookmarks.
- From the main Bible window you can open the browser from the top/right menu, or using a two-finger tap (must be enabled in Settings > General). 
- Use this feature to share a website while out in the ministry for example.

# Backups

- Backups can now be shared to any app, not just email. 
- The backup file name now includes the date and device (iPad or iPhone).

# Other Improvements 

- Added a search to bookmark folders.
- Improved support for 6.7" and mini screen sizes.
- Added the ability to lock in the screen in portrait mode in Settings > General.
- You can now change font size from the top/right menu in both the main Bible window and the Chapter window (dropping support for the pinch / zoom gesture here).

PS: iOS 14 or higher is now required to use the app.

_Equipd Bible 4.1.3 was made available on the App Store on November 20, 2023._

<hr /><em>This discussion was created from the release <a href='https://github.com/Equipd/Bible/releases/tag/v4.1.3'>Equipd Bible v4.1.3</a>.</em>

**Comment by SandyE466** [comment]
Thank you for next year's schedule. I really rely on Equipd for those schedules

**Comment by nmbgcr** [comment]
Apprezzo ancora Equipd Bibbia e le sue possibilità, personalmente noto la mancanza di una biblioteca personale all’interno della app dove si possa includere collegamenti a scritture, commenti, link a pubblicazioni web, importare immagini, video, pdf, ecc. penso che sarebbe apprezzata da molti

Translated from Google:

> I still appreciate Equipd Bible and its possibilities, I personally note the lack of a personal library within the app where you can include links to scriptures, comments, links to web publications, import images, videos, pdfs, etc. I think it would be appreciated by many

**Reply by de1mat** [reply]
Grazie mille per il tuo prezioso feedback: apprezzo molto il tempo che hai dedicato a condividerlo. Equipd Bible è progettata con un obiettivo ben preciso: rendere la consultazione e la lettura delle Scritture facile e veloce. Sebbene non includa una biblioteca personale come hai descritto, puoi aggiungere note ai versetti e queste note possono contenere link ad altre Scritture e link a siti web.

Il supporto per immagini o contenuti multimediali all'interno delle note non è attualmente disponibile e, anche se potrei prendere in considerazione qualcosa di simile in futuro, ci sono altre funzionalità più in linea con lo scopo principale dell'app a cui darei priorità. Ma il tuo contributo è prezioso e aiuta a orientare la direzione dei futuri aggiornamenti, quindi grazie ancora.
